app/src/main/java/com/coincollection/CollectionListInfo.java

```diff
@@ -456,6 +456,23 @@ void setCreationParameters(int startYear, int endYear, int mintMarkFlags, int ch
                 String.valueOf(mCheckboxFlags)};
     }
 
+    /**
+     * Get the headers for the coin CSV file
+     * @return string array with column names
+     */
+    public static String[] getCsvExportHeader() {
+        return new String[] {
+                COL_NAME,
+                COL_COIN_TYPE,
+                JSON_COL_COLLECTED,
```

Comment:
Should this have a `COL_` prefix for consistency?

Reply:
The COL_ prefix is for properties that live in the database; this one is a computed property which is exported to JSON for info only. I've renamed this to JSON_KEY_COLLECTED to be a bit clearer